tools/ml-dataprep/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-bigquery
```

Comment:
better put a concrete version number

Reply:
Done in 50e4e2428af5a29f3630985aabc7950e8a096b2a